pkg/cli/admin/ocpcertificates/regeneratemco/rotatecerts.go

```diff
@@ -120,3 +124,26 @@ func (o *RegenerateMCOOptions) Run(ctx context.Context) error {
 	}
 	return nil
 }
+
+func getServerIPsFromInfra(cfg *configv1.Infrastructure) []string {
+	if cfg.Status.PlatformStatus == nil {
+		return []string{}
+	}
+	switch cfg.Status.PlatformStatus.Type {
+	case configv1.BareMetalPlatformType:
+		return cfg.Status.PlatformStatus.BareMetal.APIServerInternalIPs
+	case configv1.OvirtPlatformType:
+		return cfg.Status.PlatformStatus.Ovirt.APIServerInternalIPs
+	case configv1.OpenStackPlatformType:
+		return cfg.Status.PlatformStatus.OpenStack.APIServerInternalIPs
+	case configv1.VSpherePlatformType:
+		if cfg.Status.PlatformStatus.VSphere.APIServerInternalIPs != nil {
```

Comment:
I know we copied this from the relevant installer code, but I don't think we need it because the nil slice will behave the same as the empty slice for us.  I think they did this conditional because they behave differently when serialized, but that's not relevant to us.